### PR TITLE
feat: ability to remove user profile picture

### LINKED ISSUE #3107: feature: Ability to remove user profile picture
## Backend

- API to delete contact conferring to the following standards 
```
DELETE api/v1/porfile/avatar
# returns head: ok  if successful 
```

## Frontend 
- User should be able to see the delete option in the profile section

Screenshot 2021-09-29 at 4 24 42 PM

- Clicking on the "Delete Avatar" button should delete the avatar.

### PR DESCRIPTION
**Pull request by santhosh-chinnasamy**
Fixes #3107

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- add spec for api
- manually tested ui functionality

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/javascript/dashboard/store/modules/auth.js [hunk 1]
```diff
@@ -118,6 +118,16 @@ export const actions = {
     }
   },
 
+  deleteAvatar: async ({ commit }) => {
+    // eslint-disable-next-line no-useless-catch
```
Comment (nithindavid) [inline app/javascript/dashboard/store/modules/auth.js:122]: We could avoid this by `throw new Error(error.message);` or `throw new Error(error);`

**Review by nithindavid — CHANGES_REQUESTED** [review]
I have a minor comment, Also could you fix the codeclimate issues also? Otgerwise it's good to merge.

File: app/mailers/conversation_reply_mailer.rb [hunk 2]
```diff
@@ -40,7 +40,9 @@ def reply_without_summary(conversation, last_queued_id)
            reply_to: reply_email,
            subject: mail_subject,
            message_id: custom_message_id,
-           in_reply_to: in_reply_to_email
+           in_reply_to: in_reply_to_email,
+           cc: cc_bcc_emails[0],
+           bcc: cc_bcc_emails[1]
```
Comment (pranavrajs) [inline app/mailers/conversation_reply_mailer.rb:45]: @santhosh-chinnasamy Was this change intentional? It looks like the change is not related to the PR.

File: db/schema.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_09_29_150415) do
+ActiveRecord::Schema.define(version: 2021_09_27_062350) do
```
Comment (pranavrajs) [inline db/schema.rb:13]: Please revert it to match the version in develop.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
I have some minor comments, please see the individual comments.

File: app/mailers/conversation_reply_mailer.rb (hunk 2 above)
Comment (santhosh-chinnasamy) [reply app/mailers/conversation_reply_mailer.rb:45]: it was linted by rubocop

File: db/schema.rb (hunk 3 above)
Comment (santhosh-chinnasamy) [reply db/schema.rb]: reverted.

File: app/javascript/dashboard/store/modules/auth.js (hunk 1 above)
Comment (santhosh-chinnasamy) [reply app/javascript/dashboard/store/modules/auth.js]: changed
